hooks/open-telemetry/pom.xml

```
@@ -5,7 +5,7 @@
     <parent>
         <groupId>dev.openfeature.contrib</groupId>
         <artifactId>parent</artifactId>
-        <version>[0.2,)</version>
+        <version>[0.2,0.3)</version>
```

```suggestion
        <version>[0.2,1.0)</version>
```
This should be safer for the future